### PR TITLE
util: update cn fisher script

### PR DESCRIPTION
**Pull request by Bluefissure**
Use `python3 get_fisher_data.py` to get fisher resources from both xivapi and [cn xivapi](https://github.com/thewakingsands/cafemaker/wiki).

### CODE REVIEW

File: util/get_fisher_data.py [hunk 1]
```diff
@@ -208,22 +209,50 @@ def append_special_place_names(places):
...
+
+
+if __name__ == "__main__":
+    example_usage = "python3 get_fisher_data.py"
+    parser = argparse.ArgumentParser(
+        description="Generate fisher data from xivapi",
+        epilog=example_usage,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    parser.add_argument('-cn', '--chs', help="If passed, get fisher data from cn xivapi", action='store_true')
```
Comment (quisquous) [inline util/get_fisher_data.py]: Can you make this always happen? I don't know why we would not want to get fisher data from xivapi?

Comment (Bluefissure) [reply util/get_fisher_data.py]: sorry for the late reply, I made this default.

**Comment by quisquous** [comment]
Thanks for making it the default.  Sorry to ask more things, but can you revert the titania-ex trigger changes and submit them in another pull request? It's easier to have this just be "cn fisher script" and I can look at triggers separately.

**Comment by Bluefissure** [comment]
> Thanks for making it the default. Sorry to ask more things, but can you revert the titania-ex trigger changes and submit them in another pull request? It's easier to have this just be "cn fisher script" and I can look at triggers separately.

Okay

**Comment by quisquous** [comment]
Thanks for updating the script.  This'll make it much easier for me to update things when patches come out.

**Pull request merged**